examples/converters.py

```
@@ -0,0 +1,116 @@
+    # In standard Python, it is use for documentation and IDE assistance purposes.
+
+    # If the conversion is successful, we will have a `discord.User` instance
+    # and can do the following:
+    user_id = user.id
+    username = user.name
+    avatar = user.avatar_url
+    await ctx.send("User found: {} -- {}\n{}".format(user_id, username, avatar))
+
+@userinfo.error
+async def userinfo_error(ctx: commands.Context, error: commands.CommandError):
+    # if the conversion above fails for any reason, it will raise `commands.BadArgument`
+    # so we handle this in this error handler:
+    if isinstance(error, commands.BadArgument):
+        return await ctx.send("Couldn't find that user.")
+
+# Custom Converter here
+class ChannelOrMemberConverter(commands.Converter):
+    async def convert(self, ctx: commands.Context, argument: str):
+        # In this example we have made a custom converter.
+        # Specifically - to check if an input is convertable to a
+        # `discord.Member` or `discord.TextChannel` instance from a
+        # single input.
+        # E.g. we have an ID we want to add to a list of ignored IDs, or similar
+
+        member_converter = commands.MemberConverter()
+        try:
+            # Try and convert to a Member instance.
+            member = await member_converter.convert(ctx, argument)
+        except commands.MemberNotFound:
+            # Could not convert to a Member instance.
+            pass
+        else:
+            # We have our `member` so lets return here.
+            return member
+
+        # Do the same for TextChannel...
+        textchannel_converter = commands.TextChannelConverter()
+        try:
+            channel = await textchannel_converter.convert(ctx, argument)
+        except commands.ChannelNotFound:
+            pass
+        else:
+            return channel
+
+        # In the case that it was not converted we should return None for expliciness' sake.
+        return None
```

i feel like it would be better to raise an error than to return `None` here